src/connected/sharedConnectedModeSettingsService.ts

```diff
@@ -110,18 +117,19 @@ export class SharedConnectedModeSettingsService implements FileSystemSubscriber
        configuration files are available to bind folder '${workspaceFolder.name}'
         to a Sonar server. Do you want to use the shared configuration?`;
     const useConfigurationHandler = async () => {
-      const quickPickItems: vscode.QuickPickItem[] = uniqueSuggestions.map(s => {
+      const quickPickItems: CustomQuickPickItem[] = uniqueSuggestions.map(s => {
         return {
           label: s.projectKey,
           description: s.organization || s.serverUrl,
-          detail: s.organization ? 'SonarQube Cloud' : 'SonarQube Server'
+          detail: s.organization ? `[${s.region}] SonarQube Cloud` : 'SonarQube Server',
```

Comment:
For consistency, is it OK to always have the region displayed here vs what is done for the tree view items?

Reply:
I added the check for Dogfooding Environment. Otherwise, in this case user needs to make an explicit choice, so the more details we display the better (I think). Also, users ending up in this branch is quite a rare case (I think) :D .